modules/cnv_install_1_4.adoc

```diff
@@ -49,9 +49,8 @@ the command.
 ====
 +
 ----
-$ ansible-playbook -i <inventory file> -e @/usr/share/ansible/kubevirt-ansible/vars/all.yml 
--e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml 
-/usr/share/ansible/kubevirt-ansible/playbooks/kubevirt.yml
+$ ansible-playbook -i <inventory_file> -e @/usr/share/ansible/kubevirt-ansible/vars/cnv.yml  
```

Comment:
I asked *not* to drop all.yml before a strict ack from QE that this actually works. no such ack was received, so why is this dropped?

even if an ack is received, I think it is important to keep it.

Reply:
I just spoke with @nellyc , she told me that as of yesterday @lukas-bednar started running CI without `all.yml`. I still think it's wrong - we should do things upstream first - but you can remove `all.yml`.